### PR TITLE
Log learning rate value in tensorboard and metrics file.

### PR DESCRIPTION
**Pull request by fhieber**
#### Pull Request Checklist ##
- [x] Changes are complete (if posting work-in-progress code, prefix your pull request title with '[WIP]'
until you can check this box.
- [x] Unit tests pass (`pytest`)
- [x] System tests pass (`pytest test/system`)
- [x] Passed code style checking (`./pre-commit.sh` or manual run of pylint & mypy)
- [x] You have considered writing a test

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tdomhan** [comment]
this is great!!

**Comment by fhieber** [comment]
I also added logging of updates/sec in the speedometer.

File: sockeye/training.py [hunk 1]
```diff
@@ -814,8 +815,10 @@ def __call__(self, epoch: int, updates: int, samples: int, tokens: int, metric:
 
         if self.init:
             if count % self.frequency == 0:
-                samples_per_sec = self.samples / (time.time() - self.tic)
-                tokens_per_sec = self.tokens / (time.time() - self.tic)
+                toc = (time.time() - self.tic)
+                updates_per_sec = self.frequency / toc
```
Comment (tdomhan) [inline sockeye/training.py]: do you mean count? frequency stays 50,no?

Comment (fhieber) [reply sockeye/training.py]: I mean frequency. toc is the time taken for going out of this block and coming back in. Hence frequency/toc is the number of updates/sec

**Pull request merged**